docs/DeveloperGuide.md

```diff
@@ -362,32 +362,58 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Use case: UC4 - Delete Fosterers**
 
-**Preconditions:** A list of fosterers is currently displayed.
-
 **MSS**
 
-1. Foster Manager requests to delete fosterers referenced by their index shown in the list.
-2. System deletes selected fosterers.
-3. System displays the updated list of fosterers.
+1. Foster Manager requests to view a list of fosterers.
+2. Foster Manager requests to delete fosterers referenced by their index shown in the list.
+3. System deletes selected fosterers.
```

Comment:
actually that's a good question .. i feel like it may be a ui detail though! I think i will omit it because the sentence where the system displays the intended output already indicates success!! (i think)

Reply:
Okay! I'll remove for mine in my next PR, will check through all the other use cases also